config.md

```diff
@@ -61,7 +61,7 @@ This specification uses the following terms:
 Here is an example image JSON file:
```

Comment:
The title for this page (up at line one) should probably be adjusted to follow the rename:

> # Image Configuration

or some such.